readme.md

```diff
@@ -3,23 +3,69 @@
-Your objective with this project is to build a web application using the 
-Mapbox API to build a map of a city of your choice.
+This project helps you gain experience in using API's to enhance the value
+your applications provide to their users. Your objective with this project is 
+to build a web application using the Mapbox API to build a map of a city of 
+your choice.
 
 ## Instructions
 
-General instructions for all Pre-Work Projects are located in the Chingu Voyage Handbook (URL posted in the
-`#read-me-first` channel on Discord). For detailed requirements checkout the ZenHub Board that is part of this repo.
+General instructions for all Solo Projects are located in the 
+[Chingu Library](https://voyage.docs.chingu.io/prework/howwork). For detailed 
+requirements checkout the following requirements.
 
-![ZenHub Screenshot](./assets/ZenHub_screenshot.png)
+### Requirements
 
-To see the ZenHub tab on the GitHub repo webpage requires that you install ZenHub's browser extension. For more
-information visit the [ZenHub Browser Extension page](https://www.zenhub.com/extension) and these 
-[installation instructions](https://t.ly/3v8w).
+*Structure*
+- [ ] Read over the Mapbox API documentation
 
-If you would like to add these tasks to your repo so you can track the progress of your Solo Project and so they will be 
-more readily available for reference you will need to manually add new issues to your ZenHub board and copy/paste the text
-in our issues to yours.
+      Things to look for in the [Mapbox API](https://docs.mapbox.com/api/) documentation:
+      - [ ] API endpoint
+      - [ ] Setting up an API token
+      - [ ] Setting up a request aka limits to the returned result, etc
+      - [ ] How to query only for the exact info you need.
 
-Good Luck!!!
+- Setup your parent component containing at least the following:
+  - [ ] Navigation (or a button to toggle the sidebar)
+  - [ ] Sidebar
+  - [ ] Map
+- Create a sidebar with the following elements:
+  - [ ] Search input field
+  - [ ] A list of search results (should match the markers on the map)
+  - [ ] A button to toggle the display/hide of the sidebar
+- Create a map component to contain the map of your city and location markers
+
+*Style*
+- [ ] You may use any style you choose. However, it should be consistent (e.g.
+font, font size, color scheme, layout, etc.).
+
+*Functionality*
+- Sidebar behaviors:
+  - [ ] By default all all available markers should be displayed
+  - [ ] Markers update in search results and on the map while the user is 
+  typing intp the search input field
+  - [ ] User should be able to click a button to toggle the sidebar visibility
+- The map component displays:
+  - [ ] The map of your city using the Mapbox API
+  - [ ] At least five markers for five unique locations or attractions in your city
+
+*Other*
+- [ ] Your repo needs to have a robust `README.md`
+- [ ] Make sure that there are no errors in the developer console before submitting
```

Comment:
Before submitting, make sure that there are no errors in the developer console

Reply:
Update applied